src/source/CppMarshal.scala

```diff
@@ -145,7 +150,7 @@ class CppMarshal(spec: Spec) extends Marshal(spec) {
     }
     def base(m: Meta): String = m match {
       case p: MPrimitive => p.cName
-      case MString => "std::string"
+      case MString => if (spec.cppUseWideStrings) "std::wstring" else "std::string"
```

Comment:
Not sure why this is showing up as my change. This came in as part of a merge I did before creating this pull request.